crypto/s2n_rsa_pss.c

```diff
@@ -183,7 +183,7 @@ static int s2n_rsa_pss_key_free(struct s2n_pkey *pkey)
 }
 
 int s2n_evp_pkey_to_rsa_pss_public_key(struct s2n_rsa_key *rsa_key, EVP_PKEY *pkey) {
-    RSA *pub_rsa_key = EVP_PKEY_get0_RSA(pkey);
+    const RSA *pub_rsa_key = EVP_PKEY_get0_RSA(pkey);
 
```

Comment:
Should there be ```POSIX_ENSURE_REF(pub_rsa_key);``` as there is in https://github.com/aws/s2n-tls/blob/061e8c696e2f15d1c230fa5a6d717892378ec739/crypto/s2n_rsa_pss.c#L196-L197